src/Microsoft.Diagnostics.EventFlow.Core/Implementations/Metadata/DataRetrievalResult.cs

```diff
@@ -12,24 +13,20 @@ public enum DataRetrievalStatus
         Success = 0,
         MetadataPropertyMissing = 1,
         InvalidMetadataValue = 2,
-        DataMissingOrInvalid = 3
+        DataMissingOrInvalid = 3,
+        InvalidMetadataType = 4
     }
 
     public class DataRetrievalResult
     {
-        private static readonly DataRetrievalResult SuccessResult = new DataRetrievalResult() { Status = DataRetrievalStatus.Success };
+        public static readonly DataRetrievalResult Success = new DataRetrievalResult() { Status = DataRetrievalStatus.Success };
 
         public DataRetrievalStatus Status { get; private set; }
         public string Message { get; private set; }
 
         // Prevent public construction
         private DataRetrievalResult() { }
 
-        public static DataRetrievalResult Success()
```

Comment:
This is surly simpler. But is this change only to make it simpler? The reason I ask here is that the other DataRetrievalResult are all methods, and this is becoming a public field, I am afraid that is a bit confusion from the caller perspective. What do you think? Or is there some other intention that I don't see?